coderd/notifications/notifier.go

```diff
@@ -230,7 +238,7 @@ func (n *notifier) prepare(ctx context.Context, msg database.AcquireNotification
 
 	helpers, err := n.fetchHelpers(ctx)
 	if err != nil {
-		return nil, xerrors.Errorf("fetch helpers: %w", err)
+		return nil, errFetchfailed
```

Comment:
`fetchHelpers` is not a very descriptive name. What you're actually doing is decorating the helpers with additional information.

```suggestion
		return nil, errDecorateHelpersFailed
```